pkgs/by-name/av/av1an-unwrapped/package.nix

```
@@ -0,0 +1,35 @@
+{ lib, rustPlatform, fetchFromGitHub, av1an-unwrapped, pkg-config, ffmpeg, nasm
+, vapoursynth, libaom, rav1e, }:
```

if you have more packages than can fit on a single line, it is best to just put each on it's own line
```suggestion
{ lib
, rustPlatform
, fetchFromGitHub
, av1an-unwrapped
, pkg-config
, ffmpeg
, nasm
, vapoursynth
, libaom
, rav1e
}:
```